src/frontend/src/components/NavBar/components/UserMenu/index.tsx

```diff
@@ -49,16 +49,16 @@ const UserMenu = ({ user }: UserMenuProps): JSX.Element => {
         <a href={ROUTES.CONTACT_US_EMAIL} target="_blank" rel="noopener">
           <MenuItem onClick={handleClose}>Contact us</MenuItem>
         </a>
-        <Link to={ROUTES.TERMS} target="_blank" rel="noopener">
+        <a href={ROUTES.TERMS} target="_blank" rel="noopener">
           <MenuItem onClick={handleClose}>Terms of Use</MenuItem>
-        </Link>
-        <Link to={ROUTES.PRIVACY} target="_blank" rel="noopener">
+        </a>
+        <a href={ROUTES.PRIVACY} target="_blank" rel="noopener">
```

Comment:
since we're opening these links in new tabs, we don't need client routing Link component